File: C/variables/exercise00.c
#include <stdio.h>

int main() {
    char serie;
    double total, seasons, number_chap, duration_chap, duration_intro;

    printf("**********Think in a serie**************\n");

    printf("Now, how much seasons have?: \n");
    scanf("%lf", &seasons);

    printf("How much chapter have?: \n");
    scanf("%lf", &number_chap);

    printf("How much minuts chapters have?: \n");
    scanf("%lf", &duration_chap);

    total = seasons * number_chap * duration_chap / 60;
    printf("You has been %lf hours look the serie\n", total);

    return 0;
}

File: python_/variables/exercise00.py

def main():
    total = None
    serie = input("What's your favorite serie?: ")
    seasons = int(input("How much seasons have?: "))
    number_chap = int(input("How much chapter have?: "))
    duration_chap = int(input("How much minuts chapters have?: "))

    total = seasons * number_chap * duration_chap / 60

    print(f'You has been {total} hours look {serie}')


if __name__ == '__main__':
    main()